### PR TITLE
Move last tests

### PR DESCRIPTION
**Pull request by huwshimi**
Move the remaining tests to the new suite and delete the old suite.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- `make test-mocha-karma` should pass

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4365/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4366/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4372/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4387/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4388/
Test PASSed.

File: Makefile [hunk 1]
```diff
@@ -67,12 +67,10 @@ help:
 	@echo "run - run the development server and watch for changes"
 	@echo "server - run the server with development settings"
 	@echo "start-karma - run Karma for development js testing"
-	@echo "mocha-test-server - run phantom for development js testing"
 	@echo "sysdeps - install the system-wide dependencies"
 	@echo "test - run python tests with the default Python"
 	@echo "test-deps - install the test dependencies"
 	@echo "test-js-karma - run newer js tests in terminal; primarily for CI build"
-	@echo "test-mocha-phantom - run older js tests that have not transitioned to karma in the terminal"
```
Comment (hatched) [inline Makefile]: YAY!

**Comment by hatched** [comment]
This is awesome! Thanks :+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4389/
Test PASSed.

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**